src/lastCharNumber.js

```diff
@@ -0,0 +1,13 @@
+const lastCharNumber = (result, button) => {
```

Comment:
I can't understand what the purpose of this function is by its name. Can you help me understand?

Reply:
@grimxreaper ^

Reply:
This is a name I planned on changing as I agree it is horrible, but it is no longer necessary as I realized I could just return result result + button in changeKeys (now updateDisplay) and replace this function all together! I thought it was doing more than it actually was.